cmd/payload-testing-prow-plugin/server.go

```diff
@@ -176,11 +184,16 @@ func (s *server) handle(l *logrus.Entry, ic github.IssueCommentEvent) string {
 		builder.spec = spec
 		var jobNames []string
 		var jobTuples []api.MetadataWithTest
+		specLogger.Debug("resolving jobs ...")
+		startResolveJobs := time.Now()
 		jobs, err := s.jobResolver.resolve(spec.ocp, spec.releaseType, spec.jobs)
+		specLogger.WithField("duration", time.Since(startResolveJobs)).Debug("resolving jobs completed")
```

Comment:
Could we log how many jobs do we have received?

Reply:
Done.